Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -942,7 +939,7 @@ - (void)callClose {
     if (self.isInitialized) {
         self.isInitialized = NO;
 
-        if ([self hasSession] && ![self.requestQueue containsClose]) {
+        if (![self.requestQueue containsClose]) {
```

Comment:
i noticed the [self hasSession] check was added recently, just wanna make sure you are not removing it here by accident.